.github/workflows/test-r-scripts.yaml

```diff
@@ -27,38 +27,46 @@ jobs:
         uses: actions/checkout@v2
         with:
           path: PACTA_analysis
+
       - name: Checkout pacta-data
         uses: actions/checkout@v2
         with:
           repository: 2DegreesInvesting/pacta-data
           token: ${{ secrets.MAURO_PAT_FOR_2DII }}
           path: pacta-data
+
       - name: Checkout create_interactive_report
         uses: actions/checkout@v2
         with:
           repository: 2DegreesInvesting/create_interactive_report
           token: ${{ secrets.MAURO_PAT_FOR_2DII }}
           path: create_interactive_report
+
       - name: Checkout StressTestingModelDev
         uses: actions/checkout@v2
         with:
           repository: 2DegreesInvesting/StressTestingModelDev
           token: ${{ secrets.MAURO_PAT_FOR_2DII }}
           path: StressTestingModelDev
+
       - name: Checkout user_results
         uses: actions/checkout@v2
         with:
           repository: 2DegreesInvesting/user_results
           token: ${{ secrets.MAURO_PAT_FOR_2DII }}
           path: user_results
 
+      - uses: r-lib/actions/setup-pandoc@master
+      - uses: r-lib/actions/setup-tinytex@v1
       - uses: r-lib/actions/setup-r@master
         with:
           r-version: ${{ matrix.config.r }}
           http-user-agent: ${{ matrix.config.http-user-agent }}
 
       - name: Install dependencies from CRAN
         run: |
+          install.packages("rmarkdown")
+          install.packages("knitr")
           install.packages("renv")
           dependencies <- renv::dependencies()$Package
           install.packages(dependencies)
```

Comment:
Since @cjyetman's review I rearranged the steps to group all R-related stuff. This change of style does not affect behaviour so I'm not asking to review again.